### PR TITLE
Ret 2048 - RET to ECM case transfer bug fix

### PR DESCRIPTION
**Pull request by btensay**
Code change to implement a bug fix for RET to ECM case transfer service. 

https://tools.hmcts.net/jira/browse/RET-2048

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by ikirsanov — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/helpers/transfertoecm/TransferToEcmCaseDataHelper.java [hunk 1]
```diff
@@ -37,11 +39,10 @@
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.CLOSED_STATE;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.POSITION_TYPE_CASE_TRANSFERRED_OTHER_COUNTRY;
 
+@Slf4j
+@Component
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/helpers/transfertoecm/TransferToEcmCaseDataHelper.java:43]: Why is this now a component?

File: src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/service/transfertoecm/CreateEcmSingleService.java [hunk 2]
```diff
@@ -22,51 +23,47 @@ public class CreateEcmSingleService {
...
-            ? officeName : transferToEcmDataModel.getOfficeCT();
+        String officeCT = transferToEcmDataModel.getOfficeCT();
         String reasonForCT = transferToEcmDataModel.getReasonForCT();
         String ccdGatewayBaseUrl = transferToEcmDataModel.getCcdGatewayBaseUrl();
         String jurisdiction = createUpdatesMsg.getJurisdiction();
         String caseId = String.valueOf(oldSubmitEvent.getCaseId());
 
-        transferNewCase(oldSubmitEvent, caseId, caseTypeId, ccdGatewayBaseUrl,
-                        jurisdiction, accessToken, reasonForCT);
+        transferNewCase(oldSubmitEvent, caseId, officeCT, ccdGatewayBaseUrl, jurisdiction, accessToken, reasonForCT);
```
[the remark this answers is not in the document]
Comment (btensay) [reply src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/service/transfertoecm/CreateEcmSingleService.java:37]: I look into it.

File: src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/helpers/transfertoecm/TransferToEcmCaseDataHelper.java (hunk 1 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/helpers/transfertoecm/TransferToEcmCaseDataHelper.java:43]: I have added one non-static method to work around the difficulty of mocking static method for the unit test that checks office name in the case transfer.

File: src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/service/transfertoecm/CreateEcmSingleService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (btensay) [reply src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/service/transfertoecm/CreateEcmSingleService.java]: Code restructured.

**Review by jthmcts — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**